### PR TITLE
Move to 4.7.0-preview1.5029 of client APIs

### PR DESCRIPTION
**Pull request by joelverhagen**
Progress on https://github.com/NuGet/Engineering/issues/1207

### CODE REVIEW

File: tests/Validation.PackageSigning.ProcessSignature.Tests/Validation.PackageSigning.ProcessSignature.Tests.csproj [hunk 1]
```diff
@@ -58,8 +58,8 @@
     <PackageReference Include="Portable.BouncyCastle">
       <Version>1.8.1.3</Version>
     </PackageReference>
-    <PackageReference Include="Test.Utility">
-      <Version>4.7.0-preview1.4948+f399fefacef3d4cc769a1580e812e1045d1d3b3e</Version>
+    <PackageReference Include="SharpZipLib">
```
Comment (joelverhagen) [inline tests/Validation.PackageSigning.ProcessSignature.Tests/Validation.PackageSigning.ProcessSignature.Tests.csproj]: Need this to force no compression when fiddling with the signature file entry. BCL seems to use the "no compression" directive just as a hint.

File: src/Validation.PackageSigning.ProcessSignature/PackageSignatureVerifierFactory.cs [hunk 2]
```diff
@@ -24,6 +24,7 @@ public static IPackageSignatureVerifier CreateMinimal()
             var settings = new SignedPackageVerifierSettings(
                 allowUnsigned: true,
                 allowUntrusted: false, // Invalid format of the signature uses this flag to determine success.
+                allowUntrustedSelfIssuedCertificate: true,
```
Comment (joelverhagen) [inline src/Validation.PackageSigning.ProcessSignature/PackageSignatureVerifierFactory.cs]: This is fine during the first pass where we catch format issues.

File: tests/Validation.PackageSigning.ProcessSignature.Tests/SignatureValidatorIntegrationTests.cs [hunk 3]
```diff
@@ -32,6 +33,8 @@ namespace Validation.PackageSigning.ProcessSignature.Tests
     [Collection(CertificateIntegrationTestCollection.Name)]
     public class SignatureValidatorIntegrationTests : IDisposable
     {
+        private const string SignatureFileName = ".signature.p7s";
```
Comment (dtivel) [inline tests/Validation.PackageSigning.ProcessSignature.Tests/SignatureValidatorIntegrationTests.cs]: Alternatively, you can use `NuGet.Packaging.Signing.SigningSpecifications.V1.SignaturePath`

Comment (joelverhagen) [reply tests/Validation.PackageSigning.ProcessSignature.Tests/SignatureValidatorIntegrationTests.cs]: Fixed.

Comment (joelverhagen) [reply tests/Validation.PackageSigning.ProcessSignature.Tests/SignatureValidatorIntegrationTests.cs]: Fixed.

**Pull request merged**